internal/ingress/controller/nginx_test.go

```diff
@@ -404,3 +406,58 @@ func TestNextPowerOf2(t *testing.T) {
 		t.Errorf("TestNextPowerOf2: expected %d but returned %d.", 0, actual)
 	}
 }
+
+func TestCleanTempNginxCfg(t *testing.T) {
+	err := cleanTempNginxCfg()
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	tmpfile, err := ioutil.TempFile("", tempNginxPattern)
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer tmpfile.Close()
+
+	dur, err := time.ParseDuration("-10m")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	oldTime := time.Now().Add(dur)
+	err = os.Chtimes(tmpfile.Name(), oldTime, oldTime)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	err = cleanTempNginxCfg()
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	tmpfile, err = ioutil.TempFile("", tempNginxPattern)
```

Comment:
This should be done before calling `cleanTempNginxCfg()`

Reply:
oops, fixed